monitor/verifier.go

```diff
@@ -47,12 +47,12 @@ func (m *EthMonitor) verifySubscribe(change sync.Change) bool {
 		&bind.CallOpts{},
 		mainchain.Hex2Addr(subscription.EthAddress))
 	if err != nil {
-		log.Errorf("Failed to query subscription desposit: %s", err)
+		log.Errorln("Failed to query subscription desposit:", err)
 		return false
 	}
 
 	if subscription.Deposit.BigInt().Cmp(deposit) != 0 {
-		log.Error("Deposit is incorrect")
+		log.Errorln("Deposit is incorrect")
```

Comment:
minor: use log.Error is fine if there is no arg provided

Reply:
It seems the main difference is `Errorln` will append a new line.

Reply:
No, our golang log func all append new line. The difference is adding spaces between args.
log.Info("hello", "world") = "helloworld"
log.Infoln("hello", "world") = "hello world"

Reply:
![image](https://user-images.githubusercontent.com/5875315/82135160-2cd0a480-97b4-11ea-828e-a2f6d9c87efe.png)

The difference between `Error` and `Errorln` is `Sprint` and `Sprintln`

Reply:
the output() add the new line, https://github.com/celer-network/goutils/blob/master/log/log.go#L207-L209
believe me, I wrote every line of the log module :)

Reply:
Ok. So why use `Sprint` and `Sprintln` accordingly?

Reply:
as I said, the difference is the space between args

Reply:
![image](https://user-images.githubusercontent.com/5875315/82135480-057bd680-97b8-11ea-8166-ba297c822d0d.png)
![image](https://user-images.githubusercontent.com/5875315/82135484-0ca2e480-97b8-11ea-8986-cf88cc1309af.png)

Ok. I read the common wrong. `Sprint` spaces are added between operands when neither is a string.